tests/StringTagFieldTest.php

```diff
@@ -120,4 +121,17 @@ protected function getNewRequest(array $parameters)
             $parameters
         );
     }
+
+    /**
+     * Test read only fields are returned
+     */
+    public function testReadonlyTransformation()
+    {
+        $record = $this->getNewStringTagFieldTestBlogPost('BlogPost2');
+        $field = new StringTagField('Tags');
+        $field->setRecord($record);
+
+        $readOnlyField = $field->performReadonlyTransformation();
+        $this->assertEquals(ReadonlyStringTagField::class, get_class($readOnlyField));
```

Comment:
You could do an assertion here on the tag string too?

Reply:
Right, I think I have added this as you wanted.

Reply:
Nice... Did that show you a bug by chance?

Reply:
Not that I recall, is there anything in particular I should be looking for?

Reply:
Haha nope, never-mind. I just noticed that the revision that added more tests also slightly tweaked the implementation and was curious if the updated test helped you find bugs. I always love it when that happens 😅 .

I don't work too closely with SS right now, so I'm not really in the position to check this out locally and play around, or fix the `composer.json`. Hopefully one of the SSLtd team or core team will get around to it soon!